src/sentry/utils/auth.py

```diff
@@ -413,11 +412,13 @@ def user_can_authenticate(self, user: User) -> bool:
         return True
 
 
-def make_login_link_with_redirect(path: str, redirect: str) -> str:
+def construct_link_with_query(path: str, query_params: dict[str, str]) -> str:
     """
     append an after login redirect to a path.
```

Comment:
This docstring's no longer accurate with the proposed changes right? Can we update it?